error: [Prusti: verification error] postcondition might not hold.
 --> $DIR/int.rs:5:11
  |
5 | #[ensures(result != 86)]
  |           ^^^^^^^^^^^^
  |
note: the error originates here
 --> $DIR/int.rs:6:1
  |
6 | / fn test1(x: i32) -> i32 {
7 | |     let y = x + 1;
8 | |     y * 2
9 | | }
  | |_^
note: counterexample for "x"
       value:   42
 --> $DIR/int.rs:6:10
  |
6 | fn test1(x: i32) -> i32 {
  |          ^
note: counterexample for "y"
       value:   43
 --> $DIR/int.rs:7:13
  |
7 |     let y = x + 1;
  |             ^^^^^
note: counterexample for "result"
       value:   86
 --> $DIR/int.rs:8:5
  |
8 |     y * 2
  |     ^^^^^

error: [Prusti: verification error] postcondition might not hold.
  --> $DIR/int.rs:12:11
   |
12 | #[ensures(result != 42)]
   |           ^^^^^^^^^^^^
   |
note: the error originates here
  --> $DIR/int.rs:13:1
   |
13 | / fn test2(x: i32) -> i32 {
14 | |     x + 21
15 | | }
   | |_^
note: counterexample for "x"
       value:   21
  --> $DIR/int.rs:13:10
   |
13 | fn test2(x: i32) -> i32 {
   |          ^
note: counterexample for "result"
       value:   42
  --> $DIR/int.rs:14:5
   |
14 |     x + 21
   |     ^^^^^^

error: aborting due to 2 previous errors

